### PR TITLE
Adds new property appendContextInErrorMessage and fluent …

### PR DESCRIPTION
**Pull request by bwiedmann**
…setter for identification purposes in error messages.

The defined row identifiers will be printed with the error message then.   

FIXES: OX-5911

### CODE REVIEW

File: src/main/java/sirius/biz/importer/format/FieldDefinition.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
     protected Supplier<String> label;
     protected Set<String> aliases = new HashSet<>();
     protected List<ValueCheck> checks = new ArrayList<>();
+    protected boolean includeIdentifierAndValueInErrorMessage;
```
Comment (idlira) [inline src/main/java/sirius/biz/importer/format/FieldDefinition.java:41]: That is a very long variable name :-)
perhaps `appendContextInErrorMessage` ?

Comment (bwiedmann) [reply src/main/java/sirius/biz/importer/format/FieldDefinition.java]: Has been renamed 👍

File: src/main/java/sirius/biz/importer/format/FieldDefinition.java [hunk 2]
```diff
@@ -264,6 +265,25 @@ public FieldDefinition markRequired() {
...
+        this.appendContextInErrorMessage = true;
+        return this;
+    }
+
+    /**
+     * Returns the info if this field's name and value should be included in error messages.
+     *
+     * @return true if this field's name and value should be included in error messages, false otherwise
+     */
+    public boolean isAppendContextInErrorMessage() {
```
Comment (idlira) [inline src/main/java/sirius/biz/importer/format/FieldDefinition.java:283]: Now this method's name look kinda strange. shouldAppend... fits better its purpose, but let's see what the others have to say.

**Comment by meg1502** [comment]
whos taking care of actually appending the field in the error message?

**Comment by idlira** [comment]
> whos taking care of actually appending the field in the error message?

In moment die Klasse, dass der Dictionary validation benutzt. zb: https://github.com/scireum/scireum-core/pull/38

File: src/main/java/sirius/biz/importer/format/FieldDefinition.java (hunk 2 above)
Comment (bwiedmann) [reply src/main/java/sirius/biz/importer/format/FieldDefinition.java]: has been renamed to shouldAppendContextInErrorMessage

**Pull request merged**